backend/app/api/events.py

```
@@ -0,0 +1,20 @@
+from app.core.database import get_db
+from app.repositories.encounter_repository import EncounterRepository
+from app.schemas.event import EventRead
+from app.services.extraction_service import ExtractionService
+
+router = APIRouter(tags=["events"])
+
+
+@router.post(
+    "/encounters/{encounter_id}/extract-events", response_model=list[EventRead]
+)
+def extract_events(encounter_id: str, db: Session = Depends(get_db)):
```

Ruff (0.16.0)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)